rllab/envs/gym_env.py

```diff
@@ -86,7 +90,10 @@ def __init__(self, env_name, record_video=True, video_schedule=None, log_dir=Non
         logger.log("observation space: {}".format(self._observation_space))
         self._action_space = convert_gym_space(env.action_space)
         logger.log("action space: {}".format(self._action_space))
-        self._horizon = env.spec.tags['wrapper_config.TimeLimit.max_episode_steps']
+        if hasattr(env.spec, 'tags') and 'wrapper_config.TimeLimit.max_episode_steps' in env.spec.tags:
+            self._horizon = env.spec.tags['wrapper_config.TimeLimit.max_episode_steps']
+        else:
+            self._horizon = 1000
```

Comment:
As far as I can tell, in updated version of gym, every env should have attribute env.max_episode_steps and this default should be unnecessary.

Reply:
Apparently the upgrade is not hard.